tests/mocks/mock-apis.ts

```diff
@@ -2,11 +2,12 @@ import { Page } from '@playwright/test';
 import { json } from '@tests/utils';
 
 import { mockStacksFeeRequests } from './mock-stacks-fees';
+import { mockMainnetTestAccountBlockstreamRequests } from './mock-utxos';
 
 export async function setupMockApis(page: Page) {
   await page.route(/chrome-extension/, route => route.continue());
   await page.route(/github/, route => route.fulfill(json({})));
   await page.route('https://api.hiro.so/', route => route.fulfill());
   await page.route('https://api.testnet.hiro.so/', route => route.fulfill());
-  await mockStacksFeeRequests(page);
+  await Promise.all([mockMainnetTestAccountBlockstreamRequests(page), mockStacksFeeRequests(page)]);
```

Comment:
Is the `Promise.all` really needed here, given it's just for the definition of the mock?  You could equally await the 5 lines above too, but haven't seen this done in docs